### ISSUE TITLE
scianimator rendering

### ISSUE THREAD

**Issue by stla** [ramnathv/poirot#18]
I have just tried a post with a scianimator animation, but the rendering of the controls bar is not very pretty: http://stla.github.io/Test_Poirot3/posts/testscianimator.html 
What should we do to get a better rendering ? My post has been generated from this source file: https://github.com/stla/Test_Poirot3/blob/gh-pages/posts/testscianimator.Rmd

**Comment by ramnathv** [comment]
I suspect it is due to CSS. I will have to take a closer look to see what is interfering. Here is `scianimator` in use with another theme, where you can see that it works well.

http://ramnathv.github.io/vistat-dupe/posts/2012-11-06-brownian-motion-with-r.html

**Comment by stla** [comment]
Yes, by modifying the html I have tried to use the same css as the one in your example : 

```
 <link rel='stylesheet' href='../libraries/frameworks/dinky/stylesheets/styles.css'> 
```

instead of 

```
 <link href="../libraries/frameworks/purus/css/bootstrap.min.css" rel="stylesheet" />
  <link href="../libraries/frameworks/purus/css/bootstrap-responsive.min.css" rel="stylesheet" />
  <link href="../libraries/frameworks/purus/css/main.css" rel="stylesheet" />
```

And then the rendering is better.

**Comment by ramnathv** [comment]
The trouble then is that the styling of the pages is dependent on Bootstrap. So you are losing that functionality, and it might lead to other inconsistencies. 

Another approach to consider is to save the animation to a separate HTML file and embed it as an iframe. You can do this with `knitr`, but will have to write some custom hooks. I use this approach a lot with rCharts, since sandboxing the js/css assets used avoids conflicts, which are very likely when you end up using a lot of libraries.

**Comment by stla** [comment]
Couldn't I do that with the `saveHTML()` function of the `animation` package ?

**Comment by ramnathv** [comment]
Yes. You should be able to do it with `saveHTML`.
